tools/download_sumo_datasets.py

```diff
@@ -244,7 +243,7 @@ def unzip_files_in_parallel(args):
 def get_zip_file_count(args):
     """Returns a tuple of four lists: all zip files, already downloaded zip files,
     remaining zip files to be downloaded, and extracted zip files."""
-    all_zip_files = args.training_input + args.training_ground_truth + args.test_input
+    all_zip_files = args.training_input + args.training_ground_truth
```

Comment:
why is the test input excluded?

Reply:
We have been storing that as a separate file.  I was unaware that it was even in there until I did this edit.  The test input and output is not currently part of these files.  This script only downloads the training data.

Reply:
Ok.